pkg/domain/infra/abi/artifact.go

```diff
@@ -124,6 +125,14 @@ func (ir *ImageEngine) ArtifactRm(ctx context.Context, opts entities.ArtifactRem
 	for _, namesOrDigest := range namesOrDigests {
 		artifactDigest, err := artStore.Remove(ctx, namesOrDigest)
 		if err != nil {
+			if opts.Ignore {
+				logrus.Debugf(
```

Comment:
this also ignores all errors which is not right as it should only ignore ErrArtifactNotExist, you chec check with errors.Is() like done in the API endpoint